coatrack/coatrack
# Feature/#73 gateway still works temporarily when admin goes down

### Instructions to test this feature

* Build the project using the root pom.xml and perform "mvn package". Building the proxy locally in the proxy folder with lead to errors.
* Start a local admin server.
* Work through the tutorial to create a service, a gateway and an API key.
* Run the new gateway and wait until the startup process is finished. In the meantime, the gateway requests all information it needs from the admin server to perform this feature.
* Shutdown the admin server.
* Perform a service call. The gateway should still direct you to the expected service API.

closes coatrack#73
